foundation/gateway/pkg/model/errors.go

```diff
@@ -0,0 +1,14 @@
+package model
+
+import "errors"
+
+// Sentinel errors for resource model operations
+var (
+	ErrLoad     = errors.New("load error")
+	ErrWatch    = errors.New("watch error")
+	ErrList     = errors.New("list error")
+	ErrConvert  = errors.New("conversion error")
```

Comment:
```suggestion
	ErrValidation  = errors.New("validation error")
```

usually used as fmt.Errorf("%w: failed to accept input due to: ....", model.ErrValidation, ....)